keyboards/kbd67/hotswap/hotswap.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 MechMerlin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef HOTSWAP_H
```

Comment:
Y U NO `#pragma once` ?

```suggestion
#pragma once
```

Reply:
Are we just doing a blanket `#pragma once` across all files?

I thought the reason for doing it was to avoid errors associated with #defines and #endifs when your headers get too complex. Is there a reason to do this when your headers are not complex?

Reply:
My understanding was we wanted this change for all header file include guards.

https://docs.qmk.fm/#/contributing?id=coding-conventions - last item on the list